api/app/signals/apps/signals/tests/test_models.py

```diff
@@ -425,6 +425,23 @@ def test_allows_contact_False_unsummited(self):
         feedback.save()
         self.assertTrue(signal.allows_contact)
 
+    @override_settings(FEATURE_FLAGS={
+        'REPORTER_MAIL_DISABLE_CONTACT_FEEDBACK_ALLOWS_CONTACT': False,
```

Comment:
See comment on `settings/base.py`. Rename feature flag to `REPORTER_MAIL_CONTACT_FEEDBACK_ALLOWS_CONTACT_ENABLED`